SuperBuild/External_LibArchive.cmake

```diff
@@ -59,6 +59,7 @@ if((NOT DEFINED LibArchive_INCLUDE_DIR
     BINARY_DIR ${EP_BINARY_DIR}
     INSTALL_DIR ${EP_INSTALL_DIR}
     CMAKE_CACHE_ARGS
+      -DCMAKE_POLICY_VERSION_MINIMUM:STRING=3.20.6
```

Comment:
This increased policy version minimum for LibArchive results in a build error that can be resolved by updating LibArchive to a newer version that includes https://github.com/libarchive/libarchive/commit/3a5a34b8211b68709090fcac71c45d9b5f59f4f2.

Reply:
Update of LibArchive contributed by @jamesobutler is available for review at:
* https://github.com/Slicer/Slicer/pull/8594